src/main/java/frc/robot/turret/TurretConstants.java

```diff
@@ -4,5 +4,6 @@
 
 public class TurretConstants {
     public static final Translation2d LOOKING_TARGET = new Translation2d(5,5);
+    public static final String LOGGING_PATH = "Turret/";
```

Comment:
add in log path constants SubsystemLogPath  = Subsystems/
and use it here

Reply:
i think that if every subsystem should start with Subsystem/<Subsystem name>/ the "Subsystem/" should be added in GBSubsystem

Reply:
you are right, can you open branch for it and merge quickly?

Reply:
yes u are right, can you open branch for it and merge quickly

Reply:
sure i'll open it from master in a sec